stable/elasticsearch-curator/templates/hooks/job.install.yaml

```
@@ -49,8 +49,15 @@ spec:
     {{- if $.Values.extraVolumeMounts }}
 {{ toYaml $.Values.extraVolumeMounts | indent 12 }}
     {{- end }}
-          command: [ "curator" ]
-          args: [ "--config", "/etc/es-curator/config.yml", "/etc/es-curator/action_file.yml" ]
+          command:
+{{ toYaml $.Values.command | indent 12 }}
+          args:
+{{- if or $.Values.dryrun $.Values.hooks.dryrun }}
```

I would rather see this conditional independent for flexibility
```suggestion
{{- if $.Values.hooks.dryrun }}
```

---

makes sense - more freedom